src/test/groovy/pl/noip/lolstats/lol/stats/jwt/JwtCheckerImplTest.groovy

```diff
@@ -30,7 +30,7 @@ class JwtCheckerImplTest extends Specification {
         timeService.millisSinceEpoch >>> [1000, 2000]
         and: "new token"
         def mail = "example@mail.com"
-        def token = jwtGenerator.generate(new Account(mail, null, null, null))
+        def token = jwtGenerator.generate(new Account(mail, "dasga", null, null))
```

Comment:
would be better to use builder here, cos if we add new field to Account, then all args constructor will change, ale we will need to change this line as well. So let's do that now only once.